Cards/CardContainer.gd

```diff
@@ -393,3 +412,18 @@ func _update_card_positions() -> void:
 		# set position and rotation
 		movement_component.state_properties.desired_position = Vector2(card_x, card_y)
 		movement_component.state_properties.desired_rotation = rotation_amount
+
+func un_queue_card(card : CardWorld):
+	set_queued_card(null)
+	_unfocus_card(card)
+	card.get_card_movement_component().set_movement_state(Enums.CardMovementState.IN_HAND)
+
+func is_queued_card_in_play_area() -> bool:
+	return get_global_mouse_position().y < play_at_height 
+
+func _handle_queued_card():
+	if(queued_card == null):
+		return
+	if(Input.is_mouse_button_pressed(2)):
```

Comment:
what is the 2 here ?

Reply:
It's for right clicking to cancel, I'll comment it at least for readability

Reply:
using a variable name would be even better